BaseListener/aws-lambda-tools-defaults.json

```diff
@@ -12,5 +12,5 @@
   "function-runtime": "dotnetcore3.1",
   "function-memory-size": 256,
   "function-timeout": 30,
-  "function-handler": "BaseListener::BaseListener.SqsFunction::FunctionHandler"
+  "function-handler": "BaseListener::BaseListener.ApiGatewayFunction::FunctionHandler"
```

Comment:
Was this branch forked from `master`?
It doesn't seem to have the dotnet 6 upgrade changes (see line 12 this file & line 4 serverless.yml file)

Reply:
the merge of `development` into your feature branch should give you all the missing changes.